opslevel/resource_opslevel_service_tag.go

```diff
@@ -238,5 +239,45 @@ func (serviceTagResource *ServiceTagResource) Delete(ctx context.Context, req re
 }
 
 func (serviceTagResource *ServiceTagResource) ImportState(ctx context.Context, req resource.ImportStateRequest, resp *resource.ImportStateResponse) {
-	resource.ImportStatePassthroughID(ctx, path.Root("id"), req, resp)
+	if !isTagValid(req.ID) {
+		resp.Diagnostics.AddError(
```

Comment:
This error added to the `resp.Diagnostics` is unhandled after it is added, lines 250-251 could cause the program to crash because of out of bounds.